libs/tree-sitter-wing/grammar.js

```diff
@@ -515,8 +515,18 @@ module.exports = grammar({
         optional($._type_annotation),
       ),
 
+    parameter_property_definition: ($) =>
```

Comment:
Yes, I don't see a scenario where other things would need it. 

What about prefixing with `initializer` instead of `ctor` as the base grammar rule is called `initializer`?

Reply:
sure, `initializer` sounds good.